### PR TITLE
Improve the UI feature of BMI Calculator Project

### PR DESCRIPTION
**Pull request by AnushaV-2002**
### 👨‍💻 Changes proposed:
1. To improve the BMI Calculator project UI feature using HTML, CSS, JAVASCRIPT
2. I have added a reset button to reset the input values and again work on it without refreshing the page 
3. Also I have shown the BMI Result.

### ✅ Check List (Check all the applicable boxes) 

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots

Screenshot (80)
Screenshot (81)

### CODE REVIEW

**Comment by NOiR-07** [comment]
You've improved the UI of the project so u can't use your name in the footer, please edit that as this project don't originally belong to you

**Comment by AnushaV-2002** [comment]
Issue #270 
Improve UI Of the project
issue resolved:-
1.My name was removed from the footer as it not belong to me I am only improved the UI of the Project. Kindly, find the changed index file.

File: Public/bmi-calculator.html [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
 <head>
-   <title>BMI Calculator</title>
-   <link rel="stylesheet" type="text/css" href="../assets/css/bmi-calculator.css">
-
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BMI Calculator</title>
+    <!--CSS Stylesheet-->
+    <link rel="stylesheet" href="./assets/CSS/bmi-calculator.css">
```
Comment (NOiR-07) [inline Public/bmi-calculator.html:9]: use ../assts/css/
as your css isn't working now

Comment (AnushaV-2002) [reply Public/bmi-calculator.html:9]: Okay @NOiR-07 I will resolve the issue and again make the pull request.

Comment (AnushaV-2002) [reply Public/bmi-calculator.html]: issue #270 
I have resolved the issue and I made the changes in file.

**Pull request merged**